src/main/java/org/mockito/ArgumentCaptor.java

```diff
@@ -114,12 +120,13 @@ public T capture() {
      * @return captured argument value
      */
     public T getValue() {
-        return this.capturingMatcher.getLastValue();
+        return capturingMatcher.getLastValue();
```

Comment:
Now that we have the `arguments`-List in this class the implementation should use it. The methods `getLastValue()` and `getValues()` can then be removed from the `CapturingMatcher` and the `ArgumentHolder` is not neccessary anymore.